src/DbBackupsTracker.cpp

```diff
@@ -162,8 +162,7 @@ bool DbBackupsTracker::isBackupRequired() const
         int backupCCN = tryGetCredentialsDbBackupChangeNumber();
```

Comment:
does this fail when the device has lower than v1.2 firmware?
is this code guaranteed to run **after** the change numbers are loaded from the device when a device is disconnected and another is plugged?

Reply:
I do test it making the daemon to return a version number lower than v1.2 but didn't test it with a real device.